sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -5,8 +5,10 @@
     <record id="view_sales_order_line_input_tree" model="ir.ui.view">
         <field name="name">sale.order.line.input.tree.</field>
         <field name="model">sale.order.line</field>
-        <field name="priority">20</field>
+        <field name="inherit_id" ref="sale.view_order_line_tree"/>
+        <field name="priority">1</field>
         <field name="arch" type="xml">
+          <tree position="replace">
```

Comment:
A replace of the whole tree? This is not very inheritable.

Reply:
It is done on purpose, respecting that all the fields of the original view exist

What would be the best alternative for this?
A long list of xpath to get to have the same?
A PR to Odoo that will never approve?

Reply:
Precisely this change is to improve the inheritance

Reply:
No, simply put a primary view and add what you need.